pkg/core/resources/model/resource.go

```diff
@@ -84,14 +84,23 @@ type ResourceList interface {
 	GetItems() []Resource
 	NewItem() Resource
 	AddItem(Resource) error
-	GetPagination() Pagination
+	GetPagination() *Pagination
```

Comment:
Just wondering, why change to pointer?

Reply:
Because I want to change the actual `Pagination` instance's `Total` value.

Reply:
Ok